R/DBI.R

```diff
@@ -208,7 +217,7 @@ setClass("DatabaseConnectorDbiResult",
 setMethod(
   "dbSendQuery",
   signature("DatabaseConnectorJdbcConnection", "character"),
-  function(conn, statement, translate = TRUE, ...) {
+  function(conn, statement, translate = FALSE, ...) {
```

Comment:
Do you think we still need the translate argument for the DBI functions? I can't think of any use cases for this, so may be cleaner to just drop it?

Reply:
Yes I think it would be cleaner to drop it actually if nothing is requiring it. I'll make the change.